libs/punchapp/src/lib/config/tableConfig.tsx

```diff
@@ -18,58 +37,164 @@ export const tableConfig: GridConfig<Punch> = {
+      field: 'Status',
+      valueGetter: (pkg) => pkg.data?.status,
       cellRenderer: (props: ICellRendererProps<Punch, string | null | undefined>) => {
-        return <DateCell dateString={props.value} />;
+        if (!props.value) {
+          return null;
+        }
+        const statusColor = hasProperty(statusColorMap, props.value)
+          ? statusColorMap[props.value]
+          : 'transparent';
+
+        return (
+          <StatusCell
+            content={props.value}
+            cellAttributeFn={() => ({ style: { backgroundColor: statusColor } })}
+          />
+        );
       },
       width: 150,
     },
     {
-      field: 'RFO Forecast/Planned',
-      valueGetter: (pkg) => pkg.data?.c07ForecastDate ?? pkg.data?.c07PlannedDate,
-      cellRenderer: (props: ICellRendererProps<Punch, string | null | undefined>) => {
-        return <DateCell dateString={props.value} />;
-      },
-      width: 150,
+      field: 'PL Sorting',
+      valueGetter: (pkg) => pkg.data?.sorting,
+      width: 100,
     },
     {
-      field: 'Formular type',
-      valueGetter: (pkg) => pkg.data?.formularType,
+      field: 'PL Type',
+      valueGetter: (pkg) => pkg.data?.type,
       width: 100,
     },
     {
-      field: 'Priority',
-      valueGetter: (pkg) => pkg.data?.priority1,
+      field: 'Estimate',
+      valueGetter: (pkg) => pkg.data?.estimate,
       width: 100,
     },
     {
-      field: 'PL Sorting',
-      valueGetter: (pkg) => pkg.data?.punchListSorting,
+      field: 'Raised by org',
+      valueGetter: (pkg) => pkg.data?.raisedBy,
+      width: 150,
+    },
+    {
+      field: 'Clearing by org',
+      valueGetter: (pkg) => pkg.data?.cleardBy,
+      width: 150,
+    },
+    {
+      field: 'Cleared',
+      valueGetter: (pkg) => pkg.data?.clearedAtDate,
+      cellRenderer: (props: ICellRendererProps<Punch, string | null | undefined>) => {
+        return props.value ? new Date(props.value).toLocaleDateString : '';
+      },
       width: 100,
     },
     {
-      field: 'PL Type',
-      valueGetter: (pkg) => pkg.data?.punchListType,
+      field: 'Verified',
+      valueGetter: (pkg) => pkg.data?.verifiedAtDate,
+      cellRenderer: (props: ICellRendererProps<Punch, string | null | undefined>) => {
+        return props.value ? new Date(props.value).toLocaleDateString : '';
+      },
       width: 100,
     },
     {
-      field: 'Estimate',
-      valueGetter: (pkg) => pkg.data?.estimate,
+      field: 'Handover plan',
+      valueGetter: (pkg) => pkg.data?.handoverPlan,
+      cellRenderer: (props: ICellRendererProps<Punch, string | null | undefined>) => {
+        return props.value ? new Date(props.value).toLocaleDateString : '';
+      },
+      width: 110,
+    },
+    {
+      field: 'Form type',
+      valueGetter: (pkg) => pkg.data?.formularType,
+      valueFormatter: (pkg) =>
+        pkg.data?.commissioningPackageId

```

Comment:
dont use commissioningPackageId and proCoSysUrls.geteCommPkgsUrl to generate form type url. Use checkListUrlId and proCoSysUrls.getFormTypeUrl instead.